src/bin/ion/commands/beta/primitive.rs

```diff
@@ -0,0 +1,74 @@
+use crate::commands::CommandConfig;
+use anyhow::{Context, Result};
+use clap::{App, Arg, ArgMatches};
+use ion_rs::binary::var_int::VarInt;
+use ion_rs::binary::var_uint::VarUInt;
+
+pub fn app() -> CommandConfig {
+    App::new("primitive")
+        .about("Prints the binary representation of an Ion encoding primitive.")
+        .arg(
+            Arg::with_name("type")
+                .short("t")
+                .required(true)
+                .takes_value(true)
+                .help("The Ion primitive encoding type. (Names are case insensitive.)")
+                .possible_values(&["VarInt", "varint", "VarUInt", "varuint"]),
+        )
+        .arg(
+            Arg::with_name("value")
+                .short("v")
+                .required(true)
+                .allow_hyphen_values(true)
+                .takes_value(true)
+                .help("The value to encode as the specified primitive."),
+        )
+}
```

Comment:
🗺️ Usage:

```
ion-beta-primitive
Prints the binary representation of an Ion encoding primitive.

USAGE:
    ion beta primitive -t <type> -v <value>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -t <type>         The Ion primitive encoding type. (Names are case insensitive.) [values: VarInt, varint, VarUInt,
                      varuint]
    -v <value>        The value to encode as the specified primitive.
```

Examples:
```
ion beta primitive -t VarInt -v 3121
hex:       18       b1
bin: 00011000 10110001

ion beta primitive -t VarInt -v -3121
hex:       58       b1
bin: 01011000 10110001

ion beta primitive -t VarUInt -v 99
hex:       e3
bin: 11100011

ion beta primitive -t VarUInt -v 99887766
hex:       2f       50       55       96
bin: 00101111 01010000 01010101 10010110
```

This command is also in `beta` and subject to change.